model/DeliveryContainerService.php

```diff
@@ -177,4 +169,44 @@ protected function getActiveFeatures(\core_kernel_classes_Resource $delivery)
             $delivery->getOnePropertyValue($this->getProperty(self::TEST_RUNNER_FEATURES_PROPERTY))
         );
     }
+
+    /**
+     * @param core_kernel_classes_Resource $delivery
+     * @return array
+     */
+    protected function getPluginsDisabledForDelivery(core_kernel_classes_Resource $delivery)
+    {
+        $enabledPlugins = [[]];
+        $disabledPlugins = [[]];
+        try {
+            $testRunnerFeatureService = $this->getServiceLocator()->get(TestRunnerFeatureService::SERVICE_ID);
+            $allTestRunnerFeatures = $testRunnerFeatureService->getAll();
+
+            if (empty($allTestRunnerFeatures)) {
+                return $disabledPlugins;
```

Comment:
`return []`
Otherwise it will return a `[[]]`

Reply:
Fixed